clients/drcachesim/tracer/tracer.cpp

```diff
@@ -1965,6 +1980,19 @@ drmemtrace_client_main(client_id_t id, int argc, const char *argv[])
                (op_record_heap.get_value() || !op_record_function.get_value().empty())) {
         FATAL("Usage error: function recording is only supported for -offline\n");
     }
+    if (op_L0_warmup_refs.get_value()) {
+
+        DR_ASSERT(!op_L0D_filter.get_value());
+        DR_ASSERT(!op_L0I_filter.get_value());
+        if (op_L0D_filter.get_value() || op_L0I_filter.get_value()) {
+            FATAL("Usage error: cannot use -L0_filter with -L0_warmup.");
+        }
+        if (op_exit_after_tracing.get_value()) {
+            FATAL("Usage error: -warmup does not currently support -exit_after_tracing.");
```

Comment:
What is the reason for not supporting?

Reply:
I had encountered some SIGSEGV crashes with `exit_after_tracing`. I have removed this check for now. We can take a look if we come across it again.